generators/client/files.js

```diff
@@ -196,7 +196,10 @@ const vueFiles = {
                 'account/reset-password/reset-password.vue',
                 'account/reset-password/reset-password.component.ts',
                 'account/settings/settings.vue',
-                'account/settings/settings.component.ts'
+                'account/settings/settings.component.ts',
+                'account/activate/activate.component.ts',
```

Comment:
can you add npm test for these new files plz ?

Reply:
You mean vue/component tests or just testing that these files are generated?

Reply:
I mean npm test with Yeoman, to be sure these files are generated

Reply:
Done in the last commit